### PR TITLE
Docs: Update the Multi-root React integration to NIM.

### PR DESCRIPTION
**Pull request by gorzelinski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Update the Multi-root React integration to NIM. Closes https://github.com/ckeditor/ckeditor5/issues/16479.

---

### Additional information

Besides updating snippets, I tried to update texts and remove build mentions.

### CODE REVIEW

File: docs/getting-started/integrations/react-multiroot.md [hunk 1]
```diff
@@ -28,22 +26,21 @@ The easiest way to use multi-root CKEditor&nbsp;5 in your React application is b
...
 ```tsx
 // App.jsx / App.tsx
 
-import React from 'react';
+import { MultiRootEditor } from 'ckeditor5';
 import { useMultiRootEditor } from '@ckeditor/ckeditor5-react';
-import MultiRootEditor from '@ckeditor/ckeditor5-build-multi-root';
 
 const App = () => {
 	const editorProps = {
```
Comment (Witoso) [inline docs/getting-started/integrations/react-multiroot.md:46]: I miss some standard plugins here. I'm not sure if this editor will run, and allow writing anything?

File: docs/getting-started/integrations/react-multiroot.md [hunk 2]
```diff
@@ -28,22 +26,21 @@ The easiest way to use multi-root CKEditor&nbsp;5 in your React application is b
...
+This guide assumes you already have a React project. If you want to create a new one, you can use the [Vite](https://vitejs.dev/guide/) CLI. It allows you to create and customize your project with templates. For example, you can set up your project with TypeScript support.
 
-Install the [CKEditor&nbsp;5 WYSIWYG editor package for React](https://www.npmjs.com/package/@ckeditor/ckeditor5-react) and the {@link getting-started/legacy-getting-started/predefined-builds#multi-root-editor multi-root editor build}. Assuming that you picked [`@ckeditor/ckeditor5-build-multi-root`](https://www.npmjs.com/package/@ckeditor/ckeditor5-build-multi-root):
+Install the [CKEditor&nbsp;5 WYSIWYG editor package for React](https://www.npmjs.com/package/@ckeditor/ckeditor5-react) and the {@link getting-started/legacy-getting-started/predefined-builds#multi-root-editor multi-root editor type}. Assuming that you picked the Multi Root Editor:
```
Comment (Witoso) [inline docs/getting-started/integrations/react-multiroot.md:31]: Change link to the editor types or examples.

**Review by Witoso — CHANGES_REQUESTED** [review]

File: docs/getting-started/integrations/react-multiroot.md (hunk 2 above)
Comment (godai78) [reply docs/getting-started/integrations/react-multiroot.md:31]: We can also start linking to editor types guide once we merge it.

**Review by godai78 — APPROVED** [review]

File: docs/getting-started/integrations/react-multiroot.md (hunk 1 above)
Comment (gorzelinski) [reply docs/getting-started/integrations/react-multiroot.md:46]: The snippet wasn't working from the start, so I assumed it was just for demo purposes. But I can make it functional - I was considering that.

**Review by Witoso — COMMENTED** [review]
If it runs, LGTM!

**Comment by gorzelinski** [comment]
> If it runs, LGTM!

Yeah, I tested it with a Vite + React sample. Thanks!

**Pull request merged**